ngx_http_upstream_fair_module.c

```diff
@@ -1175,8 +1175,7 @@ ngx_http_upstream_fair_set_session(ngx_peer_connection_t *pc, void *data)
     rc = ngx_ssl_set_session(pc->connection, ssl_session);
 
     ngx_log_debug2(NGX_LOG_DEBUG_HTTP, pc->log, 0,
-                   "set session: %p:%d",
-                   ssl_session, ssl_session ? ssl_session->references : 0);
+                   "set session: %p", ssl_session);
```

Comment:
Hello @kroeckx ,

you should also use `ngx_log_debug1()`, the number denotes the number of the arguments.